test/JDBC/input/BABEL-2967.sql

```diff
@@ -0,0 +1,12 @@
+select current_setting( 'babelfishpg_tsql.enable_metadata_inconsistency_check')
+GO
+
+
+SELECT SET_CONFIG('babelfishpg_tsql.enable_metadata_inconsistency_check','off',true)
+GO
+
+ select current_setting( 'babelfishpg_tsql.enable_metadata_inconsistency_check')
```

Comment:
please make it indentation and case aligned throughout your change

Reply:
Yes did the changes @macwoody